### PR TITLE
Fix Tarring, Add more Verbose Output to tests

### PR DESCRIPTION
**Pull request by VoR0220**
Moment of truth here.

### CODE REVIEW

File: chains/maker/archives.go [hunk 1]
```diff
@@ -33,12 +35,13 @@ func Tarball(do *definitions.Do) error {
 			return err
 		}
 		defer dir.Close()
-
+		log.Debug("Get here")
```
Comment (VoR0220) [inline chains/maker/archives.go]: Need to get rid of this

File: chains/maker/archives.go [hunk 2]
```diff
@@ -33,12 +35,13 @@ func Tarball(do *definitions.Do) error {
...
 		files, err := dir.Readdir(0) // grab the files list
 		if err != nil {
+			log.Debug("Error: ", err)
```
Comment (VoR0220) [inline chains/maker/archives.go]: and this

File: chains/maker/archives.go [hunk 3]
```diff
@@ -33,12 +35,13 @@ func Tarball(do *definitions.Do) error {
...
 			return err
 		}
-
+		log.Debug("Get here")
```
Comment (VoR0220) [inline chains/maker/archives.go]: and this

File: chains/maker/archives.go [hunk 4]
```diff
@@ -8,23 +8,25 @@ import (
...
 
 	"github.com/eris-ltd/eris/config"
 	"github.com/eris-ltd/eris/definitions"
 	"github.com/eris-ltd/eris/log"
 )
 
 func Tarball(do *definitions.Do) error {
-
-	paths, err := filepath.Glob(filepath.Join(config.ChainsPath, do.Name, "*"))
+	log.Debug("Do.name: ", do.Name)
```
Comment (VoR0220) [inline chains/maker/archives.go]: maybe get rid of this.

File: chains/maker/archives.go [hunk 5]
```diff
@@ -8,23 +8,25 @@ import (
...
+	paths, err := filepath.Glob(filepath.Join(config.ChainsPath, do.Name, strings.ToLower(do.Name)+"_*_*"))
 	if err != nil {
 		return err
 	}
+	log.Debug(paths)
```
Comment (VoR0220) [inline chains/maker/archives.go]: and this

**Comment by benjaminbollen** [comment]
FIPF

**Pull request merged**